sys/sched.c

```diff
@@ -13,7 +13,7 @@ static thread_t *idle_thread;
 static runq_t runq;
 static bool sched_active = false;
 
-#define SLICE 10
+#define SLICE 1
```

Comment:
I decided to change slice to minimum value because I want the kernel to make as much context switches as possible at this moment, so there is bigger chance for race condition, while testing I couldn't get the race condition using slice of size 10, but it was very easy to get the race condition using slice of size 1. In future when this implementation looks OK, it's good idea to just change it to longer slice,